resources/uk/gov/hmcts/gradle/Dockerfile_test

```diff
@@ -0,0 +1,8 @@
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-debug-distroless-1.3
```

Comment:
does this work on apps that are using java 8 as a compile and runtime target?

Reply:
If source and target compatibility are specified.